### DISCUSSION TITLE
如何去除Stdout输出中被设置颜色的日志中的颜色部分，例如：\u001b[32m。求解，谢谢

### DISCUSSION THREAD

**Discussion by polaris-gxf** [alibaba/ilogtail#402] [Help] [answered]
"Contents":[{"Key":"content","Value":"2022-09-16 09:03:31.013 \u001b[32mINFO \u001b[0;39m \u001b[34m[TID: N/A]\u001b[0;39m [\u001b[35mThread-30\u001b[0;39m] \u001b[36mc.s.govern.polygonsync.job.BlockTask\u001b[0;39m : 区块采集------结束------\r"},

\u001b[32m

**Comment by messixukejia** [comment]
目前没有这个能力，需要的话欢迎一起参与共建。如果有共建意向，欢迎iLogtail社区群里@徐可甲，我们可以就这个问题单独交流下，可以一起设计并实现下。

**Reply by pj1987111** [reply]
我可以提供一个process插件，格式如下，SourceKey表示要识别的key，Regex表示每个key对应的正则表达式，Replacement表示识别到正则后替换的字符串，DestKey表示正则替换后生成新的字段、原字段不变。

Type: processor_regex_replace
Fields:
  - SourceKey: content1
    Regex: '\u\w+[\d{1,3};*\d{1,3}m|test'
    Replacement: ''
  - SourceKey: content2
    Regex: (\d.*\.)\d+
    Replacement: '${1}0/16'
    DestKey: newIp

**Comment by pj1987111** [comment, marked as answer]
已支持，对应issue https://github.com/alibaba/ilogtail/issues/760
可以参考 processor_string_replace 文档进行配置
